### PR TITLE
Add skeleton

### PR DESCRIPTION
**Pull request by jaderabbit**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="vxtwitter",
+    version="0.1.0a",
+    url='http://github.com/praekelt/vumi-aat',
```
Comment (rudigiesler) [inline setup.py]: Should be http://github.com/praekelt/vumi-twitter

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Jade Abbott <jabbott@retrorabbit.co.za>
+Jeremy Thurgood <jeremy@praekelt.com>
+Justin van der Merwe <justinvdm@lavabit.com>
+Simon Cross <simon.cross@praekeltfoundation.org>
+Simon de Haan <simon@praekeltfoundation.org>
```
Comment (hodgestar) [inline AUTHORS:5]: We should maybe add Rudi here. :)

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Our dependencies are all specified in setup.py.
+
+# The following packages need to be installed from the their repositories
+# because we depend on dev versions. To upgrade, use the following command:
+#   pip install --exists-action i -U -r requirements.txt
+# This will ignore any existing repository clones. They can be updated manually
+# if desired.
+
+-e git+https://github.com/praekelt/vumi.git#egg=vumi
```
Comment (hodgestar) [inline requirements.txt]: You can install Vumi from pypi now!

Comment (hodgestar) [inline requirements.txt]: Just delete this line and add it as a dependency in `setup.py`.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    author='Praekelt Foundation',
+    author_email='dev@praekeltfoundation.org',
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[
+        'vumi',
+        # We need a dev versions of the packages above, so they have to be
+        # installed before us. They're listed first so that we fail fast
+        # instead of working through all the other requirements before
+        # discovering that they aren't available should that be the case.
```
Comment (hodgestar) [inline setup.py]: Ah, vumi is already listed. So you can just delete this comment.

File: utils/bump-version.sh [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+function inplace_sed {
+  # Note: we don't use sed -i -e ... because it isn't supported by FreeBSD
+  # sed on OS X.
+  suffix=".inplace.bak"
+  sed -i"$suffix" -e "$1" "$2"
+  rm "$2$suffix"
+}
+
+inplace_sed "s/\(version[ ]*=[ ]*[\"']\)\(.*\)\([\"'].*\)/\1${VER}\3/" setup.py
+inplace_sed "s/^\(__version__[ ]*=[ ]*[\"']\)\(.*\)\([\"'].*\)/\1${VER}\3/" vxaat/__init__.py
```
Comment (hodgestar) [inline utils/bump-version.sh]: We should update this `vxaat` reference.

**Comment by hodgestar** [comment]
Otherwise looks good.

File: .coveragerc [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  def __repr__
+  if self\.debug
+  raise AssertionError
+  raise NotImplementedError
+  if 0:
+  if __name__ == .__main__.:
+ignore_errors = True
+
+[html]
+directory = coverage-html
```
Comment (jerith) [inline .coveragerc]: Newlines at the end of files, please.

**Comment by jerith** [comment]
:+1: after adding newlines to the ends of files that don't currently have them.

**Pull request merged**